src/dev/flang/tools/docs/Html.java

```diff
@@ -50,15 +50,17 @@ public class Html extends ANY
   final DocsOptions config;
   private final Map<AbstractFeature, Map<Kind,TreeSet<AbstractFeature>>> mapOfDeclaredFeatures;
   private final String navigation;
+  private final Docs docs;
 
   /**
    * the constructor taking the options
    */
-  public Html(DocsOptions config, Map<AbstractFeature, Map<Kind,TreeSet<AbstractFeature>>> mapOfDeclaredFeatures, AbstractFeature universe)
+  public Html(DocsOptions config, Map<AbstractFeature, Map<Kind,TreeSet<AbstractFeature>>> mapOfDeclaredFeatures, AbstractFeature universe, Docs docs)
```

Comment:
It is weird to me that you pass in docs since Docs depends on HTML but it should not be the other way round. You could copy the function `allInnerAndInheritedFeatures` and pass in the SourceModule here.

Reply:
good point